<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1323</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true">MainWindow{
background-color: rgb(189, 189, 189);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_44">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1331</width>
      <height>651</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../432900212_289928807330898_532122955177928870_n.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1371</width>
      <height>821</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion Materiaux</string>
     </attribute>
     <widget class="QPushButton" name="PDF">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>490</y>
        <width>122</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton
{
border:none;
color:white;
background-color:#9dc8c2;
	
	font: 75 11pt &quot;Arial&quot;;
border-radius:10px;

}
QPushButton:Hover
{
background-color:gray;
border_radius:30px;
border-botom:2px solid gray
}
QPushButton:pressed
{
background-color:#8A8A8A;
}

</string>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="statistique">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>490</y>
        <width>122</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton
{
border:none;
color:white;
background-color:#9dc8c2;
	
	font: 75 11pt &quot;Arial&quot;;
border-radius:10px;

}
QPushButton:Hover
{
background-color:gray;
border_radius:30px;
border-botom:2px solid gray
}
QPushButton:pressed
{
background-color:#8A8A8A;
}

</string>
      </property>
      <property name="text">
       <string>Statistique</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="ajouter_p">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>60</y>
        <width>731</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>10</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(73, 187, 172);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Affichage</string>
      </property>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>691</width>
         <height>341</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">          QTableView {
                border: 2px solid #49BBAC;
                border-radius: 5px;
                background-color: #FFFFFF;
                font: 10pt &quot;Arial&quot;;
                color: black;
                selection-background-color: #49BBAC;
            }

            QTableView:focus {
                border-color: #9dc8c2;
            }

            QTableView:hover {
                border-color: #9dc8c2;
            }

            QTableView::disabled {
                background-color: #F0F0F0;
                color: #A9A9A9;
            }</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_43">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>181</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Trier selon votre choix :</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_trier_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>20</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(73, 187, 172);
    border-style: outset;
    border-width: 0px;
    border-radius: 10px;
 
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <item>
        <property name="text">
         <string>ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QUANTITE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CATEGORIE</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_tir_2">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>20</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
border:none;
color:white;
background-color:#9dc8c2;
	
	font: 75 11pt &quot;Arial&quot;;
border-radius:10px;

}
QPushButton:Hover
{
background-color:gray;
border_radius:30px;
border-botom:2px solid gray
}
QPushButton:pressed
{
background-color:#8A8A8A;
}

</string>
       </property>
       <property name="text">
        <string>Trier ðŸªœ</string>
       </property>
      </widget>
     </widget>
     <widget class="QComboBox" name="comboBox_rech_2">
      <property name="geometry">
       <rect>
        <x>672</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(73, 187, 172);
    border-style: outset;
    border-width: 0px;
    border-radius: 10px;
 
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NOM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CATEGORIE</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_42">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>-20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche par :</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>461</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(73, 187, 172);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Ajouter Materiel</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(73, 187, 172);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>id</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(73, 187, 172);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>nom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 2px solid #49BBAC;
    border-radius: 5px;
    padding: 5px;
    background-color: #FFFFFF;
    selection-background-color: #49BBAC;
    font: 8pt &quot;Arial&quot;;
    color: black;
}

QLineEdit:focus {
    border-color: #9dc8c2; /* Border color when the line edit has focus */
}

QLineEdit:hover {
    border-color: #9dc8c2; /* Border color on hover */
}

QLineEdit::disabled {
    background-color: #F0F0F0; /* Background color when disabled */
    color: #A9A9A9; /* Text color when disabled */
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 2px solid #49BBAC;
    border-radius: 5px;
    padding: 5px;
    background-color: #FFFFFF;
    selection-background-color: #49BBAC;
    font: 8pt &quot;Arial&quot;;
    color: black;
}

QLineEdit:focus {
    border-color: #9dc8c2; /* Border color when the line edit has focus */
}

QLineEdit:hover {
    border-color: #9dc8c2; /* Border color on hover */
}

QLineEdit::disabled {
    background-color: #F0F0F0; /* Background color when disabled */
    color: #A9A9A9; /* Text color when disabled */
}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_quantite">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>200</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 2px solid #49BBAC;
    border-radius: 5px;
    padding: 5px;
    background-color: #FFFFFF;
    selection-background-color: #49BBAC;
    font: 8pt &quot;Arial&quot;;
    color: black;
}

QLineEdit:focus {
    border-color: #9dc8c2; /* Border color when the line edit has focus */
}

QLineEdit:hover {
    border-color: #9dc8c2; /* Border color on hover */
}

QLineEdit::disabled {
    background-color: #F0F0F0; /* Background color when disabled */
    color: #A9A9A9; /* Text color when disabled */
}</string>
       </property>
      </widget>
      <widget class="QLabel" name="vvv">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(73, 187, 172);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>quantite</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(73, 187, 172);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>dateAjout</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_localisation">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 2px solid #49BBAC;
    border-radius: 5px;
    padding: 5px;
    background-color: #FFFFFF;
    selection-background-color: #49BBAC;
    font: 8pt &quot;Arial&quot;;
    color: black;
}

QLineEdit:focus {
    border-color: #9dc8c2; /* Border color when the line edit has focus */
}

QLineEdit:hover {
    border-color: #9dc8c2; /* Border color on hover */
}

QLineEdit::disabled {
    background-color: #F0F0F0; /* Background color when disabled */
    color: #A9A9A9; /* Text color when disabled */
}</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(73, 187, 172);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>localisation</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="le_dateAjouter">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>240</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 2px solid #49BBAC;
    border-radius: 5px;
    padding: 5px;
    background-color: #FFFFFF;
    selection-background-color: #49BBAC;
    font: 8pt &quot;Arial&quot;;
    color: black;
}

QLineEdit:focus {
    border-color: #9dc8c2; /* Border color when the line edit has focus */
}

QLineEdit:hover {
    border-color: #9dc8c2; /* Border color on hover */
}

QLineEdit::disabled {
    background-color: #F0F0F0; /* Background color when disabled */
    color: #A9A9A9; /* Text color when disabled */
}</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouter">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>310</y>
         <width>122</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
border:none;
color:white;
background-color:#9dc8c2;
	
	font: 75 11pt &quot;Arial&quot;;
border-radius:10px;

}
QPushButton:Hover
{
background-color:gray;
border_radius:30px;
border-botom:2px solid gray
}
QPushButton:pressed
{
background-color:#8A8A8A;
}

</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QComboBox" name="le_categorie">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(73, 187, 172);
    border-style: outset;
    border-width: 0px;
    border-radius: 10px;
 
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>k</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>e</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>n</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(73, 187, 172);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Categorie</string>
       </property>
      </widget>
      <widget class="QPushButton" name="modifier">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>310</y>
         <width>122</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
border:none;
color:white;
background-color:#9dc8c2;
	
	font: 75 11pt &quot;Arial&quot;;
border-radius:10px;

}
QPushButton:Hover
{
background-color:gray;
border_radius:30px;
border-botom:2px solid gray
}
QPushButton:pressed
{
background-color:#8A8A8A;
}

</string>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>490</y>
        <width>122</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton
{
border:none;
color:white;
background-color:#9dc8c2;
	
	font: 75 11pt &quot;Arial&quot;;
border-radius:10px;

}
QPushButton:Hover
{
background-color:gray;
border_radius:30px;
border-botom:2px solid gray
}
QPushButton:pressed
{
background-color:#8A8A8A;
}

</string>
      </property>
      <property name="text">
       <string>afficher </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rech_2">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>20</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(160, 214, 182);</string>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>490</y>
        <width>122</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton
{
border:none;
color:white;
background-color:#9dc8c2;
	
	font: 75 11pt &quot;Arial&quot;;
border-radius:10px;

}
QPushButton:Hover
{
background-color:gray;
border_radius:30px;
border-botom:2px solid gray
}
QPushButton:pressed
{
background-color:#8A8A8A;
}

</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_recherchepar_2">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>20</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton
{
border:none;
color:white;
background-color:#9dc8c2;
	
	font: 75 11pt &quot;Arial&quot;;
border-radius:10px;

}
QPushButton:Hover
{
background-color:gray;
border_radius:30px;
border-botom:2px solid gray
}
QPushButton:pressed
{
background-color:#8A8A8A;
}

</string>
      </property>
      <property name="text">
       <string>Recherche ðŸ”Ž </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>1341</width>
        <height>681</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>:/new/prefix1/img/bg.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="notificationTextEdit">
      <property name="geometry">
       <rect>
        <x>53</x>
        <y>440</y>
        <width>421</width>
        <height>87</height>
       </rect>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>PDF</zorder>
     <zorder>statistique</zorder>
     <zorder>ajouter_p</zorder>
     <zorder>comboBox_rech_2</zorder>
     <zorder>label_42</zorder>
     <zorder>groupBox</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>lineEdit_rech_2</zorder>
     <zorder>supprimer</zorder>
     <zorder>pushButton_recherchepar_2</zorder>
     <zorder>notificationTextEdit</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Historique</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>611</width>
        <height>501</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">          QTableView {
                border: 2px solid #49BBAC;
                border-radius: 5px;
                background-color: #FFFFFF;
                font: 10pt &quot;Arial&quot;;
                color: black;
                selection-background-color: #49BBAC;
            }

            QTableView:focus {
                border-color: #9dc8c2;
            }

            QTableView:hover {
                border-color: #9dc8c2;
            }

            QTableView::disabled {
                background-color: #F0F0F0;
                color: #A9A9A9;
            }</string>
      </property>
      <property name="title">
       <string>Ajouter Salle</string>
      </property>
      <widget class="QPushButton" name="historique">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton
{
border:none;
color:white;
background-color:#9dc8c2;
	
	font: 75 11pt &quot;Arial&quot;;
border-radius:10px;

}
QPushButton:Hover
{
background-color:gray;
border_radius:30px;
border-botom:2px solid gray
}
QPushButton:pressed
{
background-color:#8A8A8A;
}

</string>
       </property>
       <property name="text">
        <string>Notre historique</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="histo">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>551</width>
         <height>401</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">          QTableView {
                border: 2px solid #49BBAC;
                border-radius: 5px;
                background-color: #FFFFFF;
                font: 10pt &quot;Arial&quot;;
                color: black;
                selection-background-color: #49BBAC;
            }

            QTableView:focus {
                border-color: #9dc8c2;
            }

            QTableView:hover {
                border-color: #9dc8c2;
            }

            QTableView::disabled {
                background-color: #F0F0F0;
                color: #A9A9A9;
            }</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1341</width>
        <height>681</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>:/new/prefix1/img/bg.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label_4</zorder>
     <zorder>groupBox_9</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1323</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
